OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -416,7 +421,15 @@ public override bool Tick(Actor self)
 				// Only move by a full speed step if we didn't already move this tick.
 				// If we did, we limit the move to any carried-over leftover progress.
 				if (Move.lastMovePartCompletedTick < self.World.WorldTick)
-					progress += mobile.MovementSpeedForCell(mobile.ToCell);
+				{
+					var currentSpeed = mobile.MovementSpeedForCell(mobile.ToCell);
+					progress += currentSpeed;
```

Comment:
This variable isn't used anywhere else, so we can simply do `progress += mobile.MovementSpeedForCell(mobile.ToCell);` as it were before.